packages/bpk-component-graphic-promotion/src/__snapshots__/BpkGraphicPromo-test.js.snap

```diff
@@ -104,6 +106,7 @@ exports[`BpkGraphicPromo should render correctly when centre aligned 1`] = `
     <div
       aria-hidden="true"
       class="bpk-graphic-promo__container bpk-graphic-promo__container--center"
+      id="undefined__content"
```

Comment:
Should we add a snapshot test for when `contentId` has a value and is not undefined?

Reply:
Yes, I'll add one. 🙂

Reply:
Sorted. 😄